src/main/java/at/hannibal2/skyhanni/features/itemabilities/abilitycooldown/ItemAbilityCooldown.kt

```diff
@@ -75,12 +79,26 @@ object ItemAbilityCooldown {
     @HandleEvent
     fun onPlaySound(event: PlaySoundEvent) {
         when {
-            // Hyperion
+            // Necron's Blades (NECRONS_BLADE, HYPERION, SCYLLA, ASTRAEA, VALKYRIE)
             event.soundName == "mob.zombie.remedy" && event.pitch == 0.6984127f && event.volume == 1f -> {
                 val abilityScrolls = InventoryUtils.getItemInHand()?.getAbilityScrolls() ?: return
-                if (abilityScrolls.size != 3) return
-
-                ItemAbility.HYPERION.sound()
+                if (abilityScrolls.containsAll(
```

Comment:
checking `containsAll` is unnecessary. The list can only have 0-3 entries. So if there are 3 entries, it is those three and we don't need to check fi it is actually those three.